.travis.yml

```diff
@@ -1,23 +1,24 @@
-sudo: false
+sudo: required
 language: python
-addons:
-  chrome: beta
-  firefox: latest
 python:
   - "2.7"
+addons:
+  chrome: stable
+  firefox: latest
 env:
   matrix:
     - TEST_BROWSER=firefox SELENIUM_WINDOW_SIZE=1024x768
-    - TEST_BROWSER=chrome SELENIUM_WINDOW_SIZE=1024x768
     - TEST_BROWSER=firefox SELENIUM_WINDOW_SIZE=640x480
+    - TEST_BROWSER=chrome SELENIUM_WINDOW_SIZE=1024x768
     - TEST_BROWSER=chrome SELENIUM_WINDOW_SIZE=640x480
     - TEST_BROWSER=firefox-headless SELENIUM_WINDOW_SIZE=1024x768
-    - TEST_BROWSER=chrome-headless SELENIUM_WINDOW_SIZE=1024x768
     - TEST_BROWSER=firefox-headless SELENIUM_WINDOW_SIZE=640x480
+    - TEST_BROWSER=chrome-headless SELENIUM_WINDOW_SIZE=1024x768
     - TEST_BROWSER=chrome-headless SELENIUM_WINDOW_SIZE=640x480
   global:
     - DISPLAY=:99.0
 install:
+  - if [[ $TRAVIS_PYTHON_VERSION == 2* ]]; then travis_retry pip install "Django<2.0"; fi
```

Comment:
Ah! This is the piece that _is_ described in the commit message. Nothing to separate here. ;-)